t/t2262-job-list-stats.t

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+test_description='Test streaming flux job statistics RPC'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 1
+
+get_watchers() {
+	flux module stats job-list | jq -r .stats_watchers
+}
+
+test_expect_success 'create streaming job-stats script' '
+	cat >job-stats.py <<-EOT &&
+	import sys
+	import flux
+	handle = flux.Flux()
+	fut = handle.rpc("job-list.job-stats",{},flags=flux.constants.FLUX_RPC_STREAMING)
+	for i in range(int(sys.argv[1])):
+	    print(fut.get())
+	    sys.stdout.flush()
+	    fut.reset()
+	EOT
+	chmod +x job-stats.py
+'
+test_expect_success 'streaming job-stats returns one response immediately' '
+	run_timeout 30 flux python job-stats.py 1
+'
+test_expect_success 'streaming job-stats returns at least 2 responses when a job runs' '
+	flux submit sleep 1 &&
+	run_timeout 30 flux python job-stats.py 2
+'
```

Comment:
i can't help but look at this and think it's a little racy, like if the system was a little slow to launch `job-stats.py`, the sleep could finish before the job stats are run.

Maybe run job-stats.py in the background, run job to completion, and then make sure background process output 2 lines of stats?

Reply:
Yeah, let me work on that a bit.